assets/tests/components/mapPage.test.tsx

```diff
@@ -1850,7 +1850,7 @@ describe("<MapPage />", () => {
         })
       )
 
-      expect(screen.getByText("Create Detour")).toBeVisible()
+      expect(screen.getByText("Draw Detour")).toBeVisible()
```

Comment:
suggestion: do you feel like pulling this out into it's own selector too? Seems like this had to be changed in a few places as well

Reply:
Oh hey! I already had a selector that would work for this, as well as other tests, but I didn't use it universally. Updated ✔️